src/components/QrCodeForm.vue

```diff
@@ -0,0 +1,208 @@
+<script setup lang="ts">
+import { pix } from "../utils/QrCodePix";
+import { ref } from "vue";
+import { debounce } from "@/utils/debounce";
+import CustomButton from "./CustomButton.vue";
+import api from "../services/index";
+
+const props = defineProps({
+  pixTarget: String,
+  tokenValue: Number,
+});
+
+console.log(props.tokenValue);
+console.log(props.pixTarget);
+
+const qrCode = ref<string>("");
+const qrCodePayload = ref<string>("");
+const pixQrCode = pix({
+  pixKey: props.pixTarget ?? "",
+  value: props.tokenValue,
+});
+const pixIsValid = ref<number>(0);
+
+pixQrCode.base64QrCode().then((code: string) => {
+  qrCode.value = code;
+});
+
+qrCodePayload.value = pixQrCode.payload();
+
+const handleInputEvent = (event: any) => {
+  const { value } = event.target;
+
+  validatePix(value);
+};
+
+const validatePix = async (value: any) => {
+  console.log(value);
+  if(value == ''){
+    pixIsValid.value = 0;
+    return;
+  }
+  var sellerPixKey = props.pixTarget;
+  var transactionValue = props.tokenValue;
+
+  if(sellerPixKey && transactionValue){
+    var body_req = {
+      e2e_id: value,
+      pix_key: sellerPixKey,
+      pix_value: transactionValue,
+    };
+    var resp = await api.post("http://localhost:8000/validate_pix", body_req)
+      .catch((reason: Error) => {
+        console.log('entrou no erro', reason);
+        pixIsValid.value = 2;
+        return;
+      });
+    console.log("🚀 ~ file: QrCodeForm.vue:47 ~ validatePix ~ resp", resp);
+    console.log(resp.status);
+
+    pixIsValid.value = 1;
+  } else {
+    pixIsValid.value = 0;
+  }
+};
+</script>
+
+<template>
+  <div class="page">
+    <div class="text-container">
+      <span class="text font-extrabold text-2xl max-w-[30rem]">
+        Utilize o QR Code ou copie o código para realizar o Pix
+      </span>
+      <span class="text font-medium text-md max-w-[28rem]">
+        Após realizar o Pix no banco de sua preferência, insira o código de
+        autenticação para enviar a transação para a rede.
+      </span>
+    </div>
+    <div class="blur-container max-w-[28rem] text-black">
+      <div
+        class="flex-col items-center justify-center flex w-full bg-white p-8 rounded-lg"
+      >
+        <img :src="qrCode" class="w-48 h-48" />
+        <span class="text-center font-bold">Código pix</span>
+        <span class="text-center text-xs">
+          {{ pixTarget }}
```

Comment:
I think that pixTarget is incorrect, you need to show the pix payload that you get from qrCodePayload. Pix target is just the pix key, but we can talk with design team to confirm this, but I prefer to print the pix payload that allows the user to see the same thing as qrCode are showing